README.md

```diff
@@ -6,10 +6,10 @@
 
 <img src="https://github.com/salesforce/sloop/raw/master/other/sloop_logo_color_small_notext.png">
 
-----
+---
```

Comment:
Probably this helps for reads on what changed [readme.md](https://github.com/salesforce/sloop/blob/adb699f932802899573f5220491f6835377457c7/README.md) with changes

Reply:
For sure, only 3 hyphens are required so I think that's why my IDE auto formatting removed the 4th hyphen.
https://www.markdownguide.org/basic-syntax/#horizontal-rules